docs/content/intro/protocol.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Download protocol"
+date: 2018-02-11T16:58:56-05:00
+---
+
+Athens builds on top of Go CLI which specifies a set of endpoints using which it communicates with external proxies providing modules. This set of endpoints we call _Download protocol_
+
+The original vgo research paper on Download protocol can be found here: https://research.swtch.com/vgo-module
+
+Each of these endpoints sits on top of a module. Let's assume module `htp` authored by `acidburn`.
+
+So for each of endpoints mentioned bellow we will assume address `acidburn/htp/@v/{endpoint}` (e.g `acidburn/htp/@v/list`)
+
+## List of versions
+
+This endpoint returns a list of versions that Athens knows about for `acidburn/htp`. The list is just separated by newlines:
+
+```HTTP
+GET athens.io/acidburn/htp/@v/list
+```
+
+```HTML
+v1.0
+v1.1
+v2.0
+```
+
+## Version info
+
+
+```HTTP
+GET athens.io/acidburn/htp/@v/v1.0.0.info
+```
+
+This returns JSON with information about v1.0.0. It looks like this:
+
+```json
+{
+    "Name": "v1.0.0",
+    "Short": "v1.0.0",
+    "Version": "v1.0.0",
+    "Time": "1972-07-18T12:34:56Z"
+}
+```
+
+## GO.mod file
```

Comment:
Maybe `Module info`? It just looks a bit off having an all caps filename. Tricky as it's a title though.